web/src/input.rs

```diff
@@ -1,125 +1,159 @@
-        "KeyK" => KeyCode::K,
-        "KeyL" => KeyCode::L,
-        "KeyM" => KeyCode::M,
-        "KeyN" => KeyCode::N,
-        "KeyO" => KeyCode::O,
-        "KeyP" => KeyCode::P,
-        "KeyQ" => KeyCode::Q,
-        "KeyR" => KeyCode::R,
-        "KeyS" => KeyCode::S,
-        "KeyT" => KeyCode::T,
-        "KeyU" => KeyCode::U,
-        "KeyV" => KeyCode::V,
-        "KeyW" => KeyCode::W,
-        "KeyX" => KeyCode::X,
-        "KeyY" => KeyCode::Y,
-        "KeyZ" => KeyCode::Z,
-        "Semicolon" => KeyCode::SEMICOLON,
-        "Equal" => KeyCode::EQUAL,
-        "Comma" => KeyCode::COMMA,
-        "Minus" => KeyCode::MINUS,
-        "Period" => KeyCode::PERIOD,
-        "Slash" => KeyCode::SLASH,
-        "Backquote" => KeyCode::BACKQUOTE,
-        "BracketLeft" => KeyCode::LEFTBRACKET,
-        "Backslash" => KeyCode::BACKSLASH,
-        "BracketRight" => KeyCode::RIGHTBRACKET,
-        "Quote" => KeyCode::QUOTE,
-        "Numpad0" => KeyCode::NUMPAD_0,
-        "Numpad1" => KeyCode::NUMPAD_1,
-        "Numpad2" => KeyCode::NUMPAD_2,
-        "Numpad3" => KeyCode::NUMPAD_3,
-        "Numpad4" => KeyCode::NUMPAD_4,
-        "Numpad5" => KeyCode::NUMPAD_5,
-        "Numpad6" => KeyCode::NUMPAD_6,
-        "Numpad7" => KeyCode::NUMPAD_7,
-        "Numpad8" => KeyCode::NUMPAD_8,
-        "Numpad9" => KeyCode::NUMPAD_9,
-        "NumpadMultiply" => KeyCode::NUMPAD_MULTIPLY,
-        "NumpadAdd" => KeyCode::NUMPAD_ADD,
-        "NumpadSubtract" => KeyCode::NUMPAD_SUBTRACT,
-        "NumpadDecimal" => KeyCode::NUMPAD_DECIMAL,
-        "NumpadDivide" => KeyCode::NUMPAD_DIVIDE,
-        "NumpadEnter" => KeyCode::ENTER,
-        "PageUp" => KeyCode::PAGE_UP,
-        "PageDown" => KeyCode::PAGE_DOWN,
-        "End" => KeyCode::END,
-        "Home" => KeyCode::HOME,
-        "ArrowLeft" => KeyCode::LEFT,
-        "ArrowUp" => KeyCode::UP,
-        "ArrowRight" => KeyCode::RIGHT,
-        "ArrowDown" => KeyCode::DOWN,
-        "Insert" => KeyCode::INSERT,
-        "Delete" => KeyCode::DELETE,
-        "Pause" => KeyCode::PAUSE,
-        "NumLock" => KeyCode::NUM_LOCK,
-        "ScrollLock" => KeyCode::SCROLL_LOCK,
-        "F1" => KeyCode::F1,
-        "F2" => KeyCode::F2,
-        "F3" => KeyCode::F3,
-        "F4" => KeyCode::F4,
-        "F5" => KeyCode::F5,
-        "F6" => KeyCode::F6,
-        "F7" => KeyCode::F7,
-        "F8" => KeyCode::F8,
-        "F9" => KeyCode::F9,
-        "F10" => KeyCode::F10,
-        "F11" => KeyCode::F11,
-        "F12" => KeyCode::F12,
-        "F13" => KeyCode::F13,
-        "F14" => KeyCode::F14,
-        "F15" => KeyCode::F15,
-        "F16" => KeyCode::F16,
-        "F17" => KeyCode::F17,
-        "F18" => KeyCode::F18,
-        "F19" => KeyCode::F19,
-        "F20" => KeyCode::F20,
-        "F21" => KeyCode::F21,
-        "F22" => KeyCode::F22,
-        "F23" => KeyCode::F23,
-        "F24" => KeyCode::F24,
-        _ => KeyCode::UNKNOWN,
+        "Backquote" => PhysicalKey::Backquote,
```

Comment:
Just curious: I tried to figure out where this order comes from and I think it's based on the code in e.g. https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_code_values#code_values_on_windows. So why is the first not Escape?

Reply:
Oh I had no reason to use Windows-specific docs :D The order is based on the spec: https://w3c.github.io/uievents-code/#code-value-tables

First it's categorized by section, then by the key's physical location, the order should be the same as in the `PhysicalKey` enum (you can see the sections there).